media-gfx/yafaray/yafaray-3.4.1.ebuild

```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```

Comment:
Can you try with 3.7 and 3.8? Looks like they should work simply.

Reply:
Getting conflicts with blender-2.79b, the only blender version in the tree.